tools/filter_parameters.py

```diff
@@ -1,21 +1,73 @@
+import os
+
+
 def ch_length_bounds(seq, length_bounds):
-    """Ф-ия проверки соответсвия интервалу длины рида"""
+    """Function checks sequence length in read length interval"""
     if type(length_bounds) is int:
         length_bounds = [0, length_bounds]
-    return length_bounds[0] <= len(seq[0]) and len(seq[0]) <= length_bounds[1]
+        '''nuc sequence is first object in tuple 'seq' '''
+    low_bound = length_bounds[0]
+    high_bound = length_bounds[1]
+    nuc_seq = seq[0]
+    return low_bound <= len(nuc_seq) and len(nuc_seq) <= high_bound
 
 
 def ch_gc_bounds(seq, gc_bounds):
-    """Ф-ия проверки соответсвия интервалу состава GC в процентах"""
+    """Function checks of 'gc' quantity in nuc read in percents"""
+    nuc_seq = seq[0]
     if type(gc_bounds) is int:
         gc_bounds = [0, gc_bounds]
-    GC_cont = (seq[0].count('G')+seq[0].count('C'))*100/len(seq[0])
+        '''nuc sequence is first object in tuple 'seq'''
+    GC_cont = (nuc_seq.count('G') + nuc_seq.count('C')) * 100 / len(nuc_seq)
     return GC_cont >= gc_bounds[0] and GC_cont <= gc_bounds[1]
 
 
 def ch_quality_threshold(seq, quality_threshold):
-    """Ф-ия проверки соответсвия пороговому значению качества среднего рида"""
+    """Function checks that read has required quality """
     sum = 0
     for symb in seq[1]:
         sum += ord(symb)
-    return sum/len(seq[1]) >= quality_threshold+33
+        '''seq[1] - second object in tuple 'seq'''
+        quality = seq[1]
+    return sum/len(quality) >= quality_threshold+33
```

Comment:
пробелы вокруг операторов
```suggestion
    return sum / len(quality) >= quality_threshold + 33
```

-2б